sql/system.go

```diff
@@ -103,6 +103,14 @@ var (
 		keys.ZonesTableID:      privilege.ReadWriteData,
 	}
 
+	systemTables = map[string]*TableDescriptor{
```

Comment:
Makes it easier for my brain to notice that they're not references to values held elsewhere.